setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup
 
 setup(name='python-figo',
-      version='1.6.0',
+      version='1.6.1',
```

Comment:
@moritzpein good catch! Do you have an idea, where that constant could be located ideally?